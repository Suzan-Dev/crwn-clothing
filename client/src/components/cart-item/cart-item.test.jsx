import React from 'react';
import { shallow } from '../../setupTests';
import CartItem from './cart-item.component';

const mockProps = {
  imageUrl: 'www.testImage.com',
  price: 9,
  name: 'PS5',
  quantity: 2,
};
const wrapper = shallow(<CartItem item={mockProps} />);

test('should render CartItem component.', () => {
  expect(wrapper.debug()).toMatchSnapshot();
});
