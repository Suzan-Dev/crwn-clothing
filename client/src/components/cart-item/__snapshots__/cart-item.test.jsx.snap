// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CartItem component. 1`] = `
"<div className=\\"cart-item\\">
  <img src=\\"www.testImage.com\\" alt=\\"item\\" />
  <div className=\\"item-details\\">
    <div className=\\"name\\">
      PS5
    </div>
    <div className=\\"price\\">
      2
       x 
      9
    </div>
  </div>
</div>"
`;
